### PR TITLE
Implement bundle metadata API. Add a check for relevant entity.

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

File: params/url.go [hunk 1]
```diff
@@ -10,6 +10,11 @@ import (
 	"labix.org/v2/mgo/bson"
 )
 
+// IsBundle returns whether the given URL represents a bundle.
```
Comment (rogpeppe) [inline params/url.go]: s/returns/reports/

(standard phrasing)

**Comment by rogpeppe** [comment]
LGTM, thanks.

File: params/url.go (hunk 1 above)
Comment (frankban) [inline params/url.go]: Done.

**Comment by frankban** [comment]
Thanks for the review Roger.
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**